app/src/main/java/com/poklad/androidifystore/presentation/ui/screens/home/SliderAdapter.kt

```diff
@@ -0,0 +1,34 @@
+package com.poklad.androidifystore.presentation.ui.screens.home
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import com.poklad.androidifystore.R
+import com.smarteist.autoimageslider.SliderViewAdapter
+
+class SliderAdapter(private val sliderList: List<Int>) :
+    SliderViewAdapter<SliderAdapter.SliderViewHolder>() {
+
+    override fun getCount(): Int {
+        return sliderList.size
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup?): SliderAdapter.SliderViewHolder {
+        val itemView: View =
+            LayoutInflater.from(parent!!.context).inflate(R.layout.item_slider, null)
+        return SliderViewHolder(itemView)
+    }
+
+    override fun onBindViewHolder(viewHolder: SliderAdapter.SliderViewHolder, position: Int) {
+        viewHolder.bind(sliderList[position])
+    }
+
+    class SliderViewHolder(itemView: View) : ViewHolder(itemView) {
+        private val sliderIV: ImageView = itemView.findViewById(R.id.idIVSliderItem)
```

Comment:
You can use viewbinding insead of `findViewById`

Reply:
done